DA-90: Almost all requests are now checked on there response

JeannedArk opened the pull request:
If the response status code is 403 the user will be redirected to the
login page.

NicklasLinz commented:
Getting message: angular.js:12722TypeError: $rootScope.redirectToLogin is not a function

Tool then only works when you manually redirect to logIn page.

JeannedArk committed:
DA-90: Almost all requests are now checked on there response

If the response status code is 403 the user will be redirected to the
login page.

JeannedArk commented:
Upadated
